x-pack/plugins/security_solution/public/management/components/console/components/help_output.tsx

```diff
@@ -5,55 +5,30 @@
  * 2.0.
  */
 
-import React, { memo, ReactNode, useEffect, useState } from 'react';
-import { EuiCallOut, EuiCallOutProps, EuiLoadingChart } from '@elastic/eui';
-import { UserCommandInput } from './user_command_input';
-import { CommandExecutionFailure } from './command_execution_failure';
+import React, { memo, PropsWithChildren, ReactNode } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { MaybeImmutable } from '../../../../../common/endpoint/types';
+import { Command } from '..';
 import { useTestIdGenerator } from '../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../hooks/state_selectors/use_data_test_subj';
 
-export interface HelpOutputProps extends Pick<EuiCallOutProps, 'title'> {
-  input: string;
-  children: ReactNode | Promise<{ result: ReactNode }>;
-}
-export const HelpOutput = memo<HelpOutputProps>(({ input, children, ...euiCalloutProps }) => {
-  const [content, setContent] = useState<ReactNode>(<EuiLoadingChart size="l" mono />);
+type HelpOutputProps = PropsWithChildren<{
+  command: MaybeImmutable<Command>;
+  title?: ReactNode;
+}>;
+export const HelpOutput = memo<HelpOutputProps>(({ command, title, children }) => {
```

Comment:
`command` prop is not used below

Reply:
Good catch. Wonder why ESLint did not flag it 🤔

Reply:
ESLint has shut it down days too 😆